.github/workflows/ci.yml

```diff
@@ -56,6 +60,11 @@ jobs:
         path: artifacts
         key: agent-artifacts|${{ join(matrix.os, ':') }}|${{steps.rust-version.outputs.RUST_VERSION}}|${{ env.ACTIONS_CACHE_KEY_DATE }}|${{ hashFiles('src/agent/**/*') }}|${{hashFiles('src/ci/agent.sh')}}
         # note: also including the ACTIONS_CACHE_KEY_DATE to rebuild if the Prereq Cache is invalidated
+    - name: Linux Prereqs
```

Comment:
This was reordered because we need the Linux pre-reqs installed before we run `rust-prereqs.sh` in the Rust pre-reqs step (on newer versions of Ubuntu).